src/workloads/execution/BackgroundValidateCmd.yml

```diff
@@ -0,0 +1,117 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 50000
+    BatchSize: 10000
+    Document:
+      x: &x {^RandomInt: {min: 0, max: 2147483647}}
+      y: &y {^RandomInt: {min: -1000, max: 1000}}
+      z: &z {^RandomInt: {distribution: geometric, p: 0.1}}
+      a: &a {^RandomString: {length: 128}}
+      b: &b {^RandomString: {length: 15}}
+      arr:
+      - *x
+      - *y
+      - *z
+      loc2d: &loc2d [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
+      loc2dSphere: &loc2dSphere [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+    Indexes:
+    - keys: {x: 1}
+    - keys: {x: -1}
+      options: {expireAfterSeconds: 1000000}
+    - keys: {x: 1, y: 1}
+    - keys: {x: 1, y: 1, b: 1}
+    - keys: {arr: 1, z: 1}
+    - keys: {arr: 1, b : 1}
+    - keys: {a: "text"}
+    - keys: {a: 1}
+      options: {sparse: true}
+    - keys: {a: "hashed"}
+    - keys: {loc2d: "2d"}
+    - keys: {loc2dSphere: "2dsphere"}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+
+- Name: CrudWithoutValidation
+  Type: CrudActor
+  Database: *db
+  Threads: 25
+  Phases:
+  - *Nop
+  - Duration: 5 minutes
+    Collection: Collection0
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: {y: {^RandomInt: {min: -1000, max: 1000}}}
+        - WriteCommand: updateOne
+          Filter: {y: {min: -1000, max: 1000}}
+          Update: {$set: {y: {min: -1000, max: 1000}}}
+        - WriteCommand: deleteOne
+          Filter: {y: {min: -1000, max: 1000}}
+    - OperationName: find
+      OperationCommand:
+        Filter: {y: {min: -1000, max: 1000}}
+  - *Nop
+  - *Nop
+    
+- Name: CrudWithValidation
```

Comment:
Is this separate from the pervious actor so that we can get different result names for them? Okay if so, although it makes me want to ask @guoyr and @rtimmons to make it so that's no needed (assuming it is today).

Reply:
Shy of that, can we use a phase definition to avoid duplication between these two actors?

Reply:
Correct, I wanted to separate them to see the differences clearly on different graphs in Evergreen. I think using the same actor would merge them together as they're both `bulkWrite`.

Reply:
@guoyr would phase blocks here remove the duplication between CrudWithValidation and CrudWithoutValidation, while maintaining the name separation? If so, I think we should do that.

Reply:
Oh yeah. The second phase looks identical so I think we can just use a YAML anchor. ExternalPhaseConfig works but it would be overkill (because there's no need to parameterize the phase)